packages/fxa-settings/src/components/Settings/PageSettings/index.tsx

```diff
@@ -104,6 +125,14 @@ export const PageSettings = (_: RouteComponentProps) => {
         />
       </div>
       <div className="flex flex-col flex-7 max-w-full gap-8 mt-10">
+        {isInactiveVerifiedBannerVisible &&
+          {
+            /* send glean metric here. */
+          } && (
+            <div className="mx-auto">
+              <Banner {...bannerProps} />
+            </div>
+          )}
```

Comment:
This could probably do better, but I'm not sure what I could do to improve it.

Reply:
I didn't want to alter the class attributes on the Banner itself since it's a shared component, so I opted for wrapping it in another div and then setting `mx-auto` on it.